frontend/src/libs/components/input/input.tsx

```diff
@@ -31,9 +36,15 @@ const Input = <T extends FieldValues>({
 
   return (
     <label>
-      <span>{label}</span>
-      <input {...field} type={type} placeholder={placeholder} />
-      {hasError && <span>{error as string}</span>}
+      <span className={styles.inputLabel}>{label}</span>
+      <input
+        {...field}
+        className={clsx(styles.input, hasError && styles.hasError)}
+        type={type}
+        placeholder={placeholder}
+        disabled={isDisabled}
+      />
+      <div className={styles.errorMessage}>{hasError && (error as string)}</div>
```

Comment:
```suggestion
      <span className={styles.errorMessage}>{hasError && (error as string)}</span>
```
`label` cannot contain `div`. it should be a span

Reply:
But I need to specify height for errorMessage. Can I use span with display: block property?

Reply:
right

Reply:
fixed